postreise/plot/plot_upgrades_map.py

```diff
@@ -7,41 +7,42 @@
     get_dcline_differences,
 )
 from postreise.plot import colors
-from postreise.plot.plot_states import get_state_borders
-from postreise.plot.projection_helpers import project_borders, project_branch
+from postreise.plot.plot_states import plot_states
+from postreise.plot.projection_helpers import project_branch
 
 
 def _map_upgrades(
     branch_merge,
     dc_merge,
-    state_shapes,
-    b2b_indices,
+    b2b_indices=None,
     diff_threshold=100,
-    all_branch_scale=1e-3,
-    diff_branch_scale=1e-3,
+    all_branch_scale=1,
+    diff_branch_scale=0.5,
     all_branch_min=0.1,
     diff_branch_min=1.0,
     b2b_scale=5e-3,
+    figsize=(1400, 800),
+    x_range=None,
+    y_range=None,
```

Comment:
What about `background_map` to display or not the tile

Reply:
I added the broader `plot_states_kwargs` logic, covering `background_map` and other things.